amplify/backend/function/expressServer/src/models/league.model.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose')
+const {Schema} = mongoose
+
+const LeagueSchema = new mongoose.Schema(
```

Comment:
For this too can you add in one more field called "Privacy" and should be either "public" or "private".